lib/topological_inventory/ansible_tower/operations/core/ansible_tower_client.rb

```diff
@@ -120,11 +120,32 @@ def verify_ssl_mode
             default_endpoint.verify_ssl ? OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE
           end
 
+          # TODO: Join with availability_check methods
```

Comment:
:tada: I like this idea.

Reply:
yeah, after this and targeted refresh will be merged, we should do some refactoring of this. 
Maybe we could also implement single endpoint in Sources API to return everything in one request to simplify it